boot/freeldr/freeldr/arch/i386/drivemap.c

```diff
@@ -172,7 +172,7 @@ UCHAR DriveMapGetBiosDriveNumber(PCSTR DeviceName)
 
 VOID DriveMapInstallInt13Handler(PDRIVE_MAP_LIST DriveMap)
 {
-    ULONG*  RealModeIVT = (ULONG*)0x00000000;
+    ULONG*  RealModeIVT = (ULONG*)(ULONG_PTR)0x00000000;
```

Comment:
Please move the freeldr changes into a separate commit! :)

Reply:
done